src/lib.rs

```diff
@@ -53,6 +61,20 @@ mod test {
         }
     }
 
+    impl Modifier<BiggerThing> for ModifyFirst {
+        fn modify(self, bigger_thing: &mut BiggerThing) {
+            let ModifyFirst(val) = self;
+            bigger_thing.first = val;
+        }
+    }
+
+    impl Modifier<BiggerThing> for ModifySecond {
+        fn modify(self, bigger_thing: &mut BiggerThing) {
+            let ModifySecond(val) = self;
```

Comment:
Go ahead and change it if you want, it was added before tuple indexing (the `.0` thing) was added to rust.